hbase-it/src/main/java/org/apache/hadoop/hbase/chaos/factories/NoKillMonkeyFactory.java

```diff
@@ -39,11 +39,13 @@
 import org.apache.hadoop.hbase.chaos.monkies.PolicyBasedChaosMonkey;
 import org.apache.hadoop.hbase.chaos.policies.PeriodicRandomActionPolicy;
 import org.apache.hadoop.hbase.chaos.policies.TwoConcurrentActionPolicy;
+import org.apache.yetus.audience.InterfaceAudience;
 
 /**
  * Monkey factory to create a ChaosMonkey that will not need access to ssh. It will not
  * kill any services and it will not perform any restarts.
  */
+@InterfaceAudience.Private
```

Comment:
Why are the monkey factories private? Is a user not expected to call them directly?

Reply:
I think MonkeyFactories are mostly expected as a param to ChaosMonkeyRunner and are not used directly, but if you think to make them LimitedPrivate is more helpful, then I'll change this.